### PR TITLE
Test https://github.com/jenkinsci/jenkins/pull/5923 (form modernise)

### PR DESCRIPTION
**Pull request by timja**
https://github.com/jenkinsci/jenkins/pull/5923

cc @MarkEWaite you were asking how I've been testing these

### CODE REVIEW

**Comment by timja** [comment]
> plugins.WorkflowPluginTest.testSharedLibraryFromGithub
has started failing

```
Unable to locate By.cssSelector: [path='/org-jenkinsci-plugins-workflow-libs-GlobalLibraries/libraries/retriever/scm/configuredByUrlRadio[true]'] in http://127.0.0.1:61983/configure
```

**Comment by timja** [comment]
FYI @janfaracik ^^

**Comment by janfaracik** [comment]
> FYI @janfaracik ^^

Ah looks like it can't click the radiobutton, changing line 63 on `/workflow_multibranch/GithubBranchSource.java` to 
```control(by.checkbox("Repository HTTPS URL")).check();``` fixes it - not sure if there's a preferred way of selecting labels but it works 👍

**Comment by timja** [comment]
> > FYI @janfaracik ^^
> 
> Ah looks like it can't click the radiobutton, changing line 63 on `/workflow_multibranch/GithubBranchSource.java` to `control(by.checkbox("Repository HTTPS URL")).check();` fixes it - not sure if there's a preferred way of selecting labels but it works 👍

yes that's the general fix, using `by.checkbox("label name")` doesn't work so good if the same text is on the page multiple times but generally it works :)

**Comment by janfaracik** [comment]
> > > FYI @janfaracik ^^
> > 
> > 
> > Ah looks like it can't click the radiobutton, changing line 63 on `/workflow_multibranch/GithubBranchSource.java` to `control(by.checkbox("Repository HTTPS URL")).check();` fixes it - not sure if there's a preferred way of selecting labels but it works 👍
> 
> yes that's the general fix, using `by.checkbox("label name")` doesn't work so good if the same text is on the page multiple times but generally it works :)

Ah snap, I gave you the wrong line number 🤦 It should have been line 62 - this is the offending line `control("configuredByUrlRadio[true]").check();` - my bad, sorry!

**Comment by timja** [comment]
🎉  Finally passed

File: src/main/resources/ath-container/run.sh [hunk 1]
```diff
@@ -45,7 +45,8 @@ if [ ! -f $war ]; then
     case "$war" in
         "latest")
             war=jenkins-latest.war
-            url=$mirrors/war/latest/jenkins.war
+            # TODO https://github.com/jenkinsci/jenkins/pull/5923
+            url=https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.328-rc32102.8b35b58eb85d/jenkins-war-2.328-rc32102.8b35b58eb85d.war
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:49]:
```suggestion
            url=https://repo.jenkins-ci.org/snapshots/org/jenkins-ci/main/jenkins-war/2.329-SNAPSHOT/jenkins-war-2.329-20220109.205053-1.war
```

**Pull request closed without merging**